### ISSUE TITLE
Toxic exposure health description edit from VHA Digital Media                                                  

### ISSUE THREAD

**Issue by warrenjr806** [department-of-veterans-affairs/va.gov-team#57195]
## What does your team need support for? Check all that apply.
- [ ] Something else 
 
 *Give a brief description of what your team needs support with.*
Review of a change in the health service description for toxic exposures. 

Proposed language: We provide a quick screening to identify any potential exposures to toxins during your military service. The screening takes 5 to 10 minutes. **During the screening, we'll ask a series of questions and document your responses in your medical record to inform your VA health care team of any possible exposures.** We’ll also connect you to more resources if you have any concerns.

Current language: We provide a quick screening to identify any potential exposures to toxins during your military service. The screening takes 5 to 10 minutes. We’ll inform you and your VA health care team of any possible exposures. We’ll also connect you to more resources if you have any concerns.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
As soon as possible   

*Please provide an estimated date so our team can create other relevant tickets.*
  4/26/2023

## Supporting artifacts
*Please provide supporting artifacts as available.*
Provided in description

## Will this work be going through the Collaboration Cycle?
 - [ ] No

## When does this work need to be done?
- Estimated launch date:  5/1/23

## Do you plan to bring this to an upcoming content office hours session?
- [ ] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
_ VHA Digital Media 

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by RLHecht** [comment]
Going to close this issue. Please reopen if you need anything else!
